src/caselawclient/xquery/set_metadata_name.xqy

```diff
@@ -4,15 +4,27 @@ declare namespace akn = "http://docs.oasis-open.org/legaldocml/ns/akn/3.0";
 declare variable $uri as xs:string external;
 declare variable $content as xs:string external;
 
-if (fn:boolean(
+declare function local:get-name-node($root-node)
```

Comment:
I don't think this is necessary. We should be able to say `$document($uri)//akn:akomaNtoso/akn:*/akn:meta/...`, I think

Reply:
you were right, have simplified to this